src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py

```diff
@@ -22,11 +22,15 @@
 from PIL import Image
 
 from ...feature_extraction_utils import BatchFeature, FeatureExtractionMixin
-from ...file_utils import TensorType, is_pytesseract_available, requires_backends
+from ...file_utils import TensorType, is_pytesseract_available, is_torch_available, requires_backends
 from ...image_utils import ImageFeatureExtractionMixin, is_torch_tensor
 from ...utils import logging
 
 
+if is_torch_available():
+    import torch
+    from torch import nn
+
```

Comment:
Can you still import `LayoutLMv2FeatureExtractor` when you don't have torch installed?

Reply:
Can you check this @mishig25?

Reply:
Not sure if this answers your question. Added import torch lines because the newly added `post_process` method depends on torch
https://github.com/huggingface/transformers/blob/01509bf3a6851fafaa9ad0ca9deecaa05bdf6ea4/src/transformers/models/layoutlmv2/feature_extraction_layoutlmv2.py#L239-L244

Should I move the import torch statement inside `post_process` method ?

Reply:
cc'ing @LysandreJik here to check what's the best option

Reply:
I would keep the import torch statement at the top level. Moving it inside the `post_process` function means it might crash once a program is well into progress, which can be painful. I'd rather it fail early on.